pkgvars

```diff
@@ -1,6 +1,6 @@
 pkgname=citus-rebalancer
 hubproj=shard_rebalancer
 pkgdesc='Shard Rebalancer'
-pkglatest=8.0.0.citus-1
+pkglatest=8.1.0.citus-1
 releasepg=11
```

Comment:
Update them to 10,11

Reply:
I changed this line to `releasepg=10,11`

Reply:
I also checked that the travis job builds packages for both pg versions. So it's ok